modules/product/src/Plugin/Field/FieldWidget/ProductVariationAttributesWidget.php

```diff
@@ -0,0 +1,223 @@
+<?php
```

Comment:
I've reverted changes but kept this file. We have a few problems. I had some code that went further in with it, but it all went nowhere
- [ ] Variation comes back as null during validation, even if "variation" is "target_id"
- [ ] We need to set the unit price field, or else validation fails